src/App.vue

```diff
@@ -634,9 +634,6 @@ export default class App extends Mixins(
         console.log('Launch Darkly update error =', error) // eslint-disable-line no-console
       })
 
-      // set completing party before draft filing dissolution create
-      this.setCompletingParty(this.getCompletingParties())
-
```

Comment:
At this point in the flow, `isRoleStaff` is false because the auth info has not been fetched. Because of this, `getCompletingParties()` won't return the correct info for staff.

I know the comment says "before draft dissolution create" but I could find nothing in draft dissolution create code that depends on completing party, so I simply moved this code to AFTER the call to `handleDraftDissolution()` on line 643.